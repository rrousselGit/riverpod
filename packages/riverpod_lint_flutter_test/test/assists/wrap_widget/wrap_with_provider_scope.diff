Message: `Wrap with ProviderScope`
Priority: 27
Diff for file `test/assists/wrap_widget/wrap_widget.dart:11`:
```
    Map();

-     return Scaffold(
-       body: Container(),
-     );
+     return ProviderScope(child: Scaffold(
+       body: Container(),
+     ),);
  }
}
```
---
Message: `Wrap with ProviderScope`
Priority: 27
Diff for file `test/assists/wrap_widget/wrap_widget.dart:12`:
```

    return Scaffold(
-       body: Container(),
+       body: ProviderScope(child: Container(),),
    );
  }
```
---
