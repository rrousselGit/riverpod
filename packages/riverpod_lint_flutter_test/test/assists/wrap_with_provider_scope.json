[{"priority":27,"change":{"message":"Wrap with ProviderScope","edits":[{"fileStamp":0,"edits":[{"offset":281,"length":0,"replacement":",)"},{"offset":241,"length":0,"replacement":"ProviderScope(child: "}]}],"linkedEditGroups":[]}},{"priority":27,"change":{"message":"Wrap with ProviderScope","edits":[{"fileStamp":0,"edits":[{"offset":274,"length":0,"replacement":",)"},{"offset":263,"length":0,"replacement":"ProviderScope(child: "}]}],"linkedEditGroups":[]}}]