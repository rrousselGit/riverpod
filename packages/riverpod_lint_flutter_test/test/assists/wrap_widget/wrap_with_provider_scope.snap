Message: `Wrap with ProviderScope`
<<<< start: 241 -- end: 279
Scaffold(
      body: Container(),
   
==== start: 241 -- end: 302
ProviderScope(child: Scaffold(
      body: Container(),
    )
>>>>
Message: `Wrap with ProviderScope`
<<<< start: 263 -- end: 272
Container
==== start: 263 -- end: 295
ProviderScope(child: Container()
>>>>
